Command/AbstractCommand.php

```diff
@@ -652,8 +655,8 @@ private function getPhingArguments(KernelInterface $kernel, $workingDirectory, $
         $properties = array_merge(array(
             'propel.database'           => 'mysql',
             'project.dir'               => $workingDirectory,
-            'propel.output.dir'         => $kernel->getRootDir().'/propel',
-            'propel.php.dir'            => $kernel->getRootDir().'/..',
+            'propel.output.dir'         => $kernel->getProjectDir().'/var/propel',
```

Comment:
Is this change in location intentional?
`./app/propel` --> `./var/propel`

Reply:
It is, seemed better to me to group generated files together in a subdir than in the root dir.